Users/Tutorial.rst

```diff
@@ -72,8 +72,10 @@ analysis, which in this case is only the ``use_spaces`` setting. We
 recommend setting it to ``True``.
 
 coala will now check the code and, in case you use the tutorial code,
-yield two results. In the case of the SpaceConsistencyBear you will
-see trailing whitespace errors on your console.
+yield one result. SpaceConsistencyBear will detect a trailing space in
+the end of the line, after ``#include <stdio.h>`` in ``main.c`` file. coala
+will then ask you to remove the trailing space, by applying the suggested
+patch.
```

Comment:
add in brackets: (this can be done by typing 2 and then enter)

or at least i think its "2"? xD correct me if im wrong